src/main/resources/schema.sql

```diff
@@ -1,7 +1,11 @@
-CREATE TABLE IF NOT EXISTS `Order` (
-    `id`        UUID PRIMARY KEY,
-    `time`      INTEGER NOT NULL,
-    `beverage` VARCHAR(50) NOT NULL,
-    `size` VARCHAR(50) NOT NULL,
-    `sugar`        INTEGER  NOT NULL
-);
\ No newline at end of file
+-- create table "Order"
+-- (
+--     id        uuid not null primary key,
+--     time      integer,
+--     beverages varchar,
+--     size      varchar,
+--     sugar     integer
+-- );
+--
+-- alter table "Order"
+--     owner to postgres;
```

Comment:
You shouldn't push commented stuff to GIT, leave it locally to yourself. We need the working minimum, everything except it makes application more complicated and nobody wants to pay for the redundant code maintenance.